deployment/execution.js

```diff
@@ -57,23 +57,38 @@ async function runCommands(allScripts, results, args, callback) {
     let logs = ""
     var index = 0;
     var index2 = 0;
+    var httpCounter = 1;
     let protocols = Array.from( allScripts.keys() );
 
     function next() {
```

Comment:
curious why do you make this a recursive call instead of a for loop or while loop?

Reply:
The recursion causes the execution of each script to be awaited for completion before continuing on to the next execution or writing of the logs.